CHANGELOG.md

```diff
@@ -38,3 +39,4 @@ The format is described in [Contributing notes](CONTRIBUTING.md#changelog-entry-
 [@r.dubrovsky]: https://github.com/roman-dubrovsky
 [@aleks]: https://github.com/AleksSenkou
 [@ula]: https://github.com/lazycoder9
+[@theblackarrovv]: https://github.com/TheBlackArroVV/
```

Comment:
@TheBlackArroVV is it ok that you'll be marked as `theblackarrovv`? Maybe `vasya` or `vasya.kuzmik` is better?